src/sinks/splunk_hec.rs

```diff
@@ -235,6 +230,14 @@ pub fn validate_host(host: &str) -> crate::Result<()> {
     }
 }
 
+fn build_uri(mut host: &str, path: &str) -> Result<Uri, http02::uri::InvalidUri> {
+    if host.ends_with('/') {
+        host = &host[0..host.len() - 1];
+    }
```

Comment:
This is simply `let host = host.trim_end('/');` (which avoids the `mut`) and could possibly be merged into the format line below.